cinema/urls.py

```diff
@@ -8,16 +10,39 @@
     MovieViewSet,
     MovieSessionViewSet,
     OrderViewSet,
+    # MovieDetailView,
+    # MovieSessionDetailView,
```

Comment:
The `MovieDetailView` and `MovieSessionDetailView` are commented out. If these views are required for your application, you should uncomment them and ensure they are correctly implemented in your views.